packages/core/src/utils/TypeUtil.ts

```diff
@@ -1014,6 +1020,25 @@ export type Features = {
    * @default false
    */
   legalCheckbox?: boolean
+  /**
+   * @description The order of the connect methods. This is experimental and subject to change.
+   * @default ['email', 'social', 'wallet']
+   * @type {('email' | 'social' | 'wallet')[]}
+   */
+  experimental_connectMethodOrder?: ConnectMethod[]
```

Comment:
What are you thinking about adding these props in `features` object? I was not quite sure if this is the exact right place or and naming, that's why I wend with `experimental_` prefix. Do you think we should keep it or gi with `connectMethodOrder`?

Same for `walletFeaturesOrder` below. Appreciate your feedback @tomiir @svenvoskamp @magiziz @zoruka

Reply:
Mhmm good question. How about having a separate `experimental` option where we store all the experimental values ?

Reply:
But otherwise i'm ok with leaving it as is as long as it has comments

Reply:
Removed experimental prefixes - they are good to go with these names under features object I think